config/routes.rb

```diff
@@ -66,6 +72,9 @@
           post "/quizzes/:quiz_id/administer",
                to: "quizzes#administer",
                as: :quizzes_administer
+          get "/quizzes/:quiz_id/manage_students",
+              to: "quizzes#manage_students",
+              as: :quizzes_manage_students
```

Comment:
Does this not work inside of a `member` block in `resources :quizzes`?

The only reason I have the long routes is because the `member` routes don't autogenerate properly. Just want to make sure you tried the member routes first.